assets/js/blocks/active-filters/block-wrapper.tsx

```diff
@@ -11,14 +12,18 @@ import Block from './block';
 import { parseAttributes } from './utils';
 
 const BlockWrapper = ( props: Record< string, unknown > ) => {
-	const colorProps = useColorProps( props );
+	const styleProps = useStyleProps( props );
+	const parsedBlockAttributes = parseAttributes( props );
 
 	return (
 		<div
-			className={ isString( props.className ) ? props.className : '' }
-			style={ { ...colorProps.style } }
+			className={ classnames(
+				isString( props.className ) ? props.className : '',
+				styleProps.className
+			) }
+			style={ { ...styleProps.style } }
```

Comment:
:slightly_smiling_face: 

```suggestion
			style={ styleProps.style }
```